### DISCUSSION TITLE
Mamba Update moose-dev=2023.09.15

### DISCUSSION THREAD

**Discussion by milljm** [idaholab/moose#25358] [News]
A recent version of our `moose-dev` Conda package has been made available.

`mamba update --all` is usually all that is necessary to update to our latest Conda packages. Sometimes however, Conda needs that extra "just do it" kick:

```bash
mamba activate moose
mamba install moose-dev=2023.09.15
```

Afterwards, if you have not already done so, please update your MOOSE repository, as the two are always closely tied to each other.

Moving forwards, this pinned discussion will always mention the latest available `moose-dev` package version a user should be operating on.

**Comment by ykvishal** [comment]
Thank you very much for this solution. Now peacock is working smoothly on my local mac machine.
